deep_qa/training/trainer.py

```diff
@@ -56,6 +57,18 @@ class Trainer:
         Upper limit on the number of validation instances, analogous to ``max_training_instances``.
     max_test_instances: int, optional (default=None)
         Upper limit on the number of test instances, analogous to ``max_training_instances``.
+    train_steps_per_epoch: int, optional (default=None)
```

Comment:
I guess they'll be right next to each other always... I could just refer to the first one.